mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
       - AI Apps: users/ai-apps.md
       - How To: users/howto.md
       - Support: users/support.md
+      - Remote and Local pairing: developers/edge-connection.md
```

Comment:
Have you tested this locally? The file is now in users directory, but the link in mkdocs.yml points to a file in the dev dir.